<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurSubjWidget</class>
 <widget class="QWidget" name="CurSubjWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="_idLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="_idLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_nameLabel">
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="_nameLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="_ageLabel">
     <property name="text">
      <string>年龄</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="_ageLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="_yearLabel">
     <property name="text">
      <string>岁</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="_sexLabel">
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="_sexLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="_heightLabel">
     <property name="text">
      <string>身高</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="_heightLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="_cmLabel">
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="_weightLabel">
     <property name="text">
      <string>体重</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="_weightLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="_kgLabel">
     <property name="text">
      <string>Kg</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
